src/rgw/support/rgw_support_bundle

```diff
@@ -81,6 +82,13 @@ class RGWSupportBundle:
         else:
             Log.error("RGW log file does not exists hence skipping log file collection.")
 
+        # copy addb logs
+        addb_log_path = os.path.join(log_dir, 'addb_files-*')
```

Comment:
whats the log_dir value here. and why cant we copy dir where addb files are be present ?

Add some examples plz. 
Also add plugin is not enabled yet, so with this PR merge even, make sure no issue with support bundle generation is seen.

Reply:
log_dir is mentioned in line 74 above 
` log_dir = os.path.join(log_base, f'rgw/{machine_id}') `
 even if  the plugin is unavailable , the code wont break anything as 
 `addb_dirs = glob.glob(addb_log_path)` will return an empty list [] 
        and no opertaion will take place as nothing to iterate in for loop

Reply:
> and why cant we copy dir where addb files are be present ?

what about this. 
also addb files will be scattered in '/etc/cortx/log/rgw/{machine_id}' dir ?

Reply:
> > and why cant we copy dir where addb files are be present ?
> 
> what about this. also addb files will be scattered in '/etc/cortx/log/rgw/{machine_id}' dir ?

in last sprint we created addb directory as a part of our mini-provisioner deployment. (https://github.com/Seagate/cortx-rgw-integration/pull/77)
hence addb directory we should use is " /etc/cortx/log/rgw/{machine-id}/addb_files-{motr fid value}/" and inside this we are expecting addb files to be generated.

Reply:
@cdeshmukh  depicted the same in the demo above
create 2 files each in for testing as mentioned by @sachitanands 

'/etc/cortx/log/rgw/e6d3c4dcb0574a27aa218003c7f14a6b/addb_files-0x7200000000000001:0x30':
total 0
-rw-r--r-- 1 root root 0 May 11 07:41 1_addb.file
-rw-r--r-- 1 root root 0 May 11 07:41 2_addb.file

'/etc/cortx/log/rgw/e6d3c4dcb0574a27aa218003c7f14a6b/addb_files-0x7300000000000001:0x30':
total 0
-rw-r--r-- 1 root root 0 May 11 07:42 11_abbd.file
-rw-r--r-- 1 root root 0 May 11 07:42 12_abbd.file

Reply:
"addb_files-{motr fid value}" is a directory